library/src/main/kotlin/org/codice/compliance/Common.kt

```diff
@@ -118,11 +118,11 @@ fun Log.debugWithSupplier(message: () -> String) {
  * @param name - Name of Assertions.children
  * @return list of Assertions.children matching the name provided
  */
-fun Node.children(name: String): List<Node> {
+fun Node.children(name: String = "default"): List<Node> {
     val childNodes = mutableListOf<Node>()
     for (i in (this.childNodes.length - 1) downTo 0) {
         this.childNodes.item(i).apply {
-            if (localName == name) childNodes.add(this)
+            if (localName == name || name == "default") childNodes.add(this)
```

Comment:
Open to ideas on how to make this cleaner. I want to provide the option of not passing in the `name` parameter and just returning all children.

Reply:
I like this much better. Putting this fix in my next PR 👍